geode-core/src/main/java/org/apache/geode/internal/cache/EntrySnapshot.java

```diff
@@ -123,6 +123,20 @@ public Object getRawValue(boolean forceCopy) {
     return v;
   }
 
+  /**
+   * If the value is available as a CachedDeserializable instance then return it.
+   * Otherwise behave as if getValue() was called.
+   */
+  public Object getValuePreferringCachedDeserializable() {
+    checkEntryDestroyed();
+    Object v = this.regionEntry.getValue(null);
```

Comment:
Please rename variable to something more meaningful other than the character `v`

Reply:
done